packages/seacas/applications/epu/EP_ExodusEntity.h

```diff
@@ -193,39 +184,35 @@ namespace Excn {
   class CommunicationMap
   {
   public:
-    CommunicationMap() : id(0), entityCount(0), type('U') {}
```

Comment:
You shouldn't even need to state the default constructor here, due to the member initializers below.

Reply:
That is why it is being removed.... ;)